torch_xla/csrc/runtime/runtime.cpp

```diff
@@ -24,13 +24,14 @@ InitializeComputationClient() {
   // Ref: https://github.com/pytorch/xla/pull/8267
   //
   // static bool use_ifrt = sys_util::GetEnvBool("XLA_USE_IFRT", false);
-  static const bool use_ifrt = false;
+  const bool use_ifrt = false;
   if (sys_util::GetEnvString(env::kEnvPjRtDevice, "") != "") {
     auto* client =
         (use_ifrt)
             ? static_cast<ComputationClient*>(new IfrtComputationClient())
             : static_cast<ComputationClient*>(new PjRtComputationClient());
     g_computation_client_initialized = true;
+    ABSL_CHECK(client);
```

Comment:
FYI, this is unnecessary as `new` guarantees to either return a non-null pointer or crash.